app/main.py

```diff
@@ -11,3 +11,10 @@
 #     print(Actor.objects.all())
 #     Actor.objects.delete(1)
 #     print(Actor.objects.all())
+from models import Actor
```

Comment:
It's a good practice to keep all your imports at the top of your file. This makes it easier to see what modules your script is using.